### PR TITLE
chore: see also for fern ui and reportr #6

### PR DESCRIPTION
**Pull request by muralicharan**
#6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,3 +44,7 @@ Welcome to the Fern Project, an innovative open-source solution designed to enha
     RunSpecs(t, "Adder Suite", Label("this-is-a-suite-level-label","anotherFlag"))
   ```
     The fern report will have the test report along with the labels.
+
+**See Also**
+fern-ui: `https://github.com/Guidewire/fern-ui`
```
Comment (anoop2811) [inline README.md:49]: Perhaps just have:
1. Fern UI (as a hyperlink): <a short discription>
2. Fern Ginkgo Reporter (as hyperlink): <a short discription>

File: README.md [hunk 2]
```diff
@@ -44,3 +44,7 @@ Welcome to the Fern Project, an innovative open-source solution designed to enha
     RunSpecs(t, "Adder Suite", Label("this-is-a-suite-level-label","anotherFlag"))
   ```
     The fern report will have the test report along with the labels.
+
+**See Also**
```
Comment (anoop2811) [inline README.md:48]: You might want to check the README in your preview of the markdown. It doesnt look correct

File: README.md (hunk 1 above)
Comment (muralicharan) [reply README.md:49]: thanks for the comment @anoop2811 , makes sense, will make the change

File: README.md (hunk 2 above)
Comment (muralicharan) [reply README.md:48]: My bad, agree, will fix.

**Comment by muralicharan** [comment]
@anoop2811 I fixed PR comments, checked preview, looks good. Please check, thanks.

**Review by anoop2811 — APPROVED** [review]

**Pull request merged**